test/modules/obsagent-client/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "obsagent-client"
+version = "0.1.0"
+authors = ["Noel Campbell <noelcampbell@microsoft.com>"]
```

Comment:
I think it is usally the edge team:
https://github.com/Azure/iotedge/blob/master/test/modules/generic-mqtt-tester/Cargo.toml#L4